zerver/models.py

```diff
@@ -975,7 +975,7 @@ class UserProfile(AbstractBaseUser, PermissionsMixin):
     AVATAR_FROM_USER = u'U'
     AVATAR_SOURCES = (
         (AVATAR_FROM_GRAVATAR, 'Hosted by Gravatar'),
-        (AVATAR_FROM_USER, 'Uploaded by user'),
+        (AVATAR_FROM_USER, 'Uploaded by user')
```

Comment:
That comma was there for a reason -- to make it easy to add additional values without changing existing lines.  Can you add it back?

Reply:
sorry, I've added another `AVATAR_SOURCE` but then we decided it wasn't needed, when I removed it I probably removed the comma by mistake :/